app/controllers/questions/adopted_child_controller.rb

```diff
@@ -2,10 +2,8 @@ module Questions
   class AdoptedChildController < QuestionsController
     include AuthenticatedClientConcern
 
-    def self.show?(intake)
-      intake.had_dependents_yes?
-    end
+    def self.show?(intake) = false
```

Comment:
these two controllers seem to be intentionally not removed, but rather changing show to false. what is the logic behind that? why not delete them?

Reply:
I thought about this yesterday and I think I'm leaning towards just disabling because I am reluctant to remove too much on my first ticket on unfamiliar code that isn't well socialized among the team. If you feel _really_ strongly that I should fully remove controllers, remove tests, remove forms, and remove views I might be persuaded, but I currently am preferring a light touch.

Reply:
Good question here. I normally love-love-love tossing out code, but the light touch might be better here, for a couple reasions: this feels like an unusual situation: a bunch of last-minute changes that could, conceivably, be reversed again in next year's form -- or, heaven forbid, even reversed in some sort of 11th-hour update in the next few weeks (which is unlikely but not impossible). and also, keeping the per-ticket surface area of change small will help us meet the tight deadline we have. that said, i am open to discussing further 😄

Reply:
thanks to you both for explaining!